packages/rest/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.tgz
+lib*
+package
```

Comment:
Is it necessary to have package-local `.gitignore` file? Aren't our packages "inheriting" the same global `.gitignore` settings?

Reply:
This came as a result of direct migration from core to rest.
AFAIK, it's been put in each of the packages.
https://github.com/strongloop/loopback-next/blob/master/packages/core/.gitignore

Reply:
Interesting. Let's keep consistency then.